### PR TITLE
Support Electra Protocol network

### PR DESCRIPTION
**Pull request by sandrocsimas**

### CODE REVIEW

File: lib/src/models/network.dart [hunk 1]
```diff
@@ -525,15 +525,15 @@ class OmniXepNetwork implements BasedUtxoNetwork {
...
-      "Omni XEP network does not support P2WPKH/P2WSH");
+  String get p2wpkhHrp => conf.params.p2wpkhHrp!;
 
   /// Checks if the current network is the mainnet.
   @override
   bool get isMainnet => true;
 
   @override
   final List<BitcoinAddressType> supportedAddress = const [
+    SegwitAddresType.p2wpkh,
```
Comment (GorbulasDev) [inline lib/src/models/network.dart:536]: Missing an "s" in address.

Comment (sandrocsimas) [reply lib/src/models/network.dart:536]: This is the existing Enum

Comment (sandrocsimas) [reply lib/src/models/network.dart:536]: https://github.com/mrtnetwork/bitcoin_base/blob/main/lib/src/bitcoin/address/core.dart

Comment (mrtnetwork) [reply lib/src/models/network.dart:536]: > https://github.com/mrtnetwork/bitcoin_base/blob/main/lib/src/bitcoin/address/core.dart

hi, you must add config in ```blockchain_utils``` for ```BIP-44``` and ```BIP-89``` confing, or if we have any testnet i can testing  and add all of them to the package.

Comment (sandrocsimas) [reply lib/src/models/network.dart:548]: @mrtnetwork, done!

Comment (sandrocsimas) [reply lib/src/models/network.dart:548]: @mrtnetwork I've added BIP-44, BIP-84, and the test net.

**Pull request merged**